docs/howto/hooks/lua.md

```diff
@@ -420,6 +420,25 @@ The `value` string should be in [ISO8601](https://en.wikipedia.org/wiki/ISO_8601
 
 Returns a new 128-bit [RFC 4122 UUID](https://www.rfc-editor.org/rfc/rfc4122){: target="_blank" } in string representation.
 
+### `net/url`
+
+Provides a `parse` function that parses a raw url into a URL structure.
```

Comment:
Parse URL string into parts, returns a table with the URL's host, path, scheme, query and fragment.

Reply:
👍